added_mass/proposal.md

```diff
@@ -0,0 +1,255 @@
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>
+
+# Hydrodynamic Added Mass Proposal
+
+* **Authors**:
+Louise Poubel `<louise@openrobotics.org>`,
+Andrew Hamilton `<hamilton@mbari.org>`,
+Michael Anderson `<anderson@mbari.org>`
+* **Status**: Draft
+* **SDFormat Version**: 1.9
+* **`libsdformat` Version**: 12.X
+
+## Introduction
+
+This proposal suggests the addition of elements under the `//link/inertial` element
+to support the inclusion of [hydrodynamic added mass] (https://en.wikipedia.org/wiki/Added_mass)
+effects in SDFormat 1.9.
+
+Current implementations, such as
+[gz::sim::systems::Hydrodynamics](https://gazebosim.org/api/gazebo/6.9/classignition_1_1gazebo_1_1systems_1_1Hydrodynamics.html#details),
+include forces due to added mass effects by estimating these forces from the last time-steps 
+acceleration value and applying these estimates as separate external forces, reducing simulation accuracy and stability.
+
+Native support of added mass through the SDF specification makes it possible for physics
+engines to handle these terms together with other inertial terms during the physics update
+step.
+
+## Document summary
+
+The proposal includes the following sections:
+
+* [Motivation](#motivation): An explanation of the background and rationale behind the proposal
+* [Proposed changes](#proposed-changes): Terms to be added to the SDFormat specification and C++ API
+
+## Motivation
+
+Newton's second law describes how forces affect a body's motion in an inertial frame. It can be written as:
+
+$$ (\bf{M} + \bf{\mu})   \ddot{\bf x} = \sum{F({\bf x}, t)} $$
+
+where $M$ is the body mass inertia matrix, $\mu$ is the hydrodynamic added mass matrix, 
+$\sum{F}$ is the sum of all forces applied to the body, and $\ddot{\bf x}$ is the resulting acceleration, with:
+
+$$
+    {\bf x}^T 
+    =
+    \begin{bmatrix}
+      x           & y           & z           & p           &  q         & r 
+    \end{bmatrix}
+$$
+
+where,
+
+* $x$ : Position in X axis
```

Comment:
@hamilton8415 , I know it's a bit odd, but the previous style (`\\(..\\)`) is the only one I got to render correctly for inline code with MathJax. It [explicitly disallows](https://docs.mathjax.org/en/latest/input/tex/delimiters.html) `$..$` inline by default. You can see this is not being displayed correctly on the preview link. 

```suggestion
* \\(x\\): Position in X axis
```

Reply:
I went ahead and fixed all instances of this